app/src/pages/CounterPage/containers/CounterPageContainers.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+
+import Counter from "../components/Counter";
+
+class CounterPageContainers extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      countValue: 0,
+      parityValue: "Введено нулевое число",
+    };
+  }
+
+  handleIncrement = () => {
+    this.setState((state) => {
+      const incrementedValue = state.countValue + 1;
+      if (incrementedValue > -1) {
+        return {
+          countValue: incrementedValue,
+        };
+      }
+    });
+  };
+
+  handleDecrement = () => {
+    this.setState((state) => {
+      const decrementedValue = state.countValue - 1;
+      if (decrementedValue > -1) {
+        return {
+          countValue: decrementedValue,
+        };
+      }
+    });
```

Comment:
setState method run every time. Try to rewrite your logic in the following manner: if -> run setState method else do nothing